mission_control/test/test_fixed_rudder_behavior.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+"""
+ * Software License Agreement (BSD License)
+ *
+ *  Copyright (c) 2020, QinetiQ, Inc.
+ *  All rights reserved.
+ *
+ *  Redistribution and use in source and binary forms, with or without
+ *  modification, are permitted provided that the following conditions
+ *  are met:
+ *
+ *   * Redistributions of source code must retain the above copyright
+ *     notice, this list of conditions and the following disclaimer.
+ *   * Redistributions in binary form must reproduce the above
+ *     copyright notice, this list of conditions and the following
+ *     disclaimer in the documentation and/or other materials provided
+ *     with the distribution.
+ *   * Neither the name of QinetiQ nor the names of its
+ *     contributors may be used to endorse or promote products derived
+ *     from this software without specific prior written permission.
+ *
+ *  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ *  "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ *  LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+ *  FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+ *  COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ *  INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+ *  BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ *  LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ *  CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+ *  LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ *  ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ *  POSSIBILITY OF SUCH DAMAGE.
+"""
+import sys
+import os
+import unittest
+import math
+import rosnode
+import rospy
+import rostest
+from mission_control.msg import ReportExecuteMissionState
+from mission_control.msg import FixedRudder
+from load_execute_mission import LoadExecuteMission
+
+
+class TestFixedRudderBehavior(unittest.TestCase):
+    """
+        The test:
+        -   Load a fixed rudder mission.
+        -   execute the mission
+        -   Wait until the mission is complete (behavior_time)
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        rospy.init_node('test_fixed_rudder_behavior')
+
+    def setUp(self):
+        mission_execute_state = ReportExecuteMissionState.PAUSED
+        self.report_execute_mission = None
+        self.fixed_rudder_goal = []
+        self.mission = LoadExecuteMission()
+        self.fixed_rudder_goal = FixedRudder()
+
+        # Subscribers
+        self.exec_state_sub = rospy.Subscriber('/mission_control_node/report_mission_execute_state',
+                                               ReportExecuteMissionState, self.callback_mission_execute_state)
```

Comment:
@patapufate nit: maybe the helper class can track mission state?

Reply:
Yes. you're right. We will use it in all the tests.